mapper/pchain/tx_parser.go

```diff
@@ -384,16 +385,22 @@ func (t *TxParser) insToOperations(
 			Type: metaType,
 		}
 
-		if transferInput, ok := in.In.(*secp256k1fx.TransferInput); ok {
-			metadata.SigIndices = transferInput.SigIndices
+		input := in.In
+		if stakeableIn, ok := input.(*stakeable.LockIn); ok {
```

Comment:
we do something similar with outputs, not sure why we would process inputs differently

Reply:
lgtm. @bilgehansahin this should resolve the missing locked-ins bug.